src/DIRAC/WorkloadManagementSystem/Agent/VirtualMachineMonitorAgent.py

```diff
@@ -159,7 +159,7 @@ def execute(self):
 
         # Get load
         avgLoad, avgRequiredSamples = self.__getLoadAvg()
-        self.log.info("Load Average is %.2f" % avgLoad)
+        self.log.info("Load Average", f"{avgLoad:.2f}")
```

Comment:
This, and other py3 only fixes can't be in v7r3 branch.

Reply:
I tried and it failed, as we also run it with python 2

Reply:
for example 
```
E     File "/home/runner/work/DIRAC/DIRAC/src/DIRAC/WorkloadManagementSystem/DB/TaskQueueDB.py", line 299
E       self.log.error("Failed to insert condition", f"{field} {result['Message']}")
E                                                                                 ^
E   SyntaxError: invalid syntax
```